lms/static/scripts/postmessage_json_rpc/server/methods.js

```diff
@@ -14,3 +14,28 @@ export function requestConfig() {
   const clientConfigObj = JSON.parse(configEl.textContent).hypothesisClient;
   return clientConfigObj;
 }
+
+/**
+ * Return the groups for the Hypothesis client to show.
+ */
+export async function requestGroups() {
+  const configObj = JSON.parse(
+    document.querySelector('.js-config').textContent
+  );
+
+  function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+  }
+
+  if (configObj.dev === true) {
```

Comment:
super dumb question. do we really need "dev" flag? I'm just thinking ahead here, does it even make sense to get the client code calling `requestGroups` before its done? I'm sure its useful to have perhaps a client branch that makes requests to `requestGroups` but I'm just not sure its valuable to put that into master branch before its done. Was the intent here we get the client making the groups call, then backfill the requestGroups with a real method since the sync/ api (or similar is done) ?

Reply:
Yes, let's get the client work done before the LMS work is done. That will allow us to parallelize the client work and the LMS work, and so release the sections feature sooner. And let's actually merge that client PR into master and release it, so the whole work of reviewing/merging/releasing the client changes can be done in parallel with ongoing LMS changes.

The `requestGroups` method in this PR _is a real method_ that really does return the list of one course group. It will later be changed to return a list of more than one section groups.